### PR TITLE
Adding temporary ping command for role reminder

### PR DESCRIPTION
**Pull request by jess-sys**

### CODE REVIEW

File: cogs/Admin.py [hunk 1]
```diff
@@ -166,6 +166,26 @@ def switch(value):
...
+    @commands.command(pass_context=True, hidden=True)
+    @commands.check(is_admin)
+    async def pingrole(self, context):
+        pinged_user_count = 0
+        graduation_roles = ["2018", "2019", "2020", "2021", "2022", "2023"]
+        special_roles = ["Pedagogy", "Pedagogy Assistant", "Admin", "Administrateur", "Staff"]
+        for member in context.message.server.members:
+            flag = 2
+            for mrole in member.roles:
+                if mrole.name in graduation_roles or mrole.name in self.bot.rank_whitelist:
```
Comment (Tina-otoge) [inline cogs/Admin.py]: Use more pythonic way to check if two arrays match instead of using (ugly) nested conditionals

File: cogs/Admin.py [hunk 2]
```diff
@@ -166,6 +166,26 @@ def switch(value):
...
+                    flag -= 1
+                if mrole.name in special_roles:
+                    flag = 0
+            if flag > 0:
+                    pinged_user_count += 1
+                    await self.bot.send_message(member, "**EPITECH 2022 Server - Announcement**\n\nDon't forget to ask for city and graduation role in EPITECH 2022 Server.\nYou have to ask them before September, 2nd 11:42 PM.\n\n"
```
Comment (Tina-otoge) [inline cogs/Admin.py]: Use Discord.Embed, you can look for examples in the Info cog

**Review by Tina-otoge — CHANGES_REQUESTED** [review]
Please refactor conditionals and use Discord.Embed

**Review by jess-sys — COMMENTED** [review]
Did the needed improvements.

**Review by Tina-otoge — APPROVED** [review]
Please stop using variables that live for literally 2 lines

**Pull request merged**